### DISCUSSION TITLE
Combining a Quarto Website with Auto-Generated Documentation

### DISCUSSION THREAD

**Discussion by BradyAJohnston** [quarto-dev/quarto-cli#4925] [Q&A] [answered]
Hi all, I'm wanting to include auto-generated documentation from `.py` files (like readthedocs)` in combination with a Quarto website. I have a website with a bunch of documentation already for a project (https://bradyajohnston.github.io/MolecularNodes/) but I am wanting to include some pages on python  modules that are included as part of the project. 

Does anyone know of a way to just generate some `.md` files from `.py` files, that could then be included in a Quarto website?

**Comment by mcanouil** [comment, marked as answer]
Like this?
<https://pypi.org/project/quartodoc/>

**Reply by BradyAJohnston** [reply]
I think this might be exactly what I am after. Wasn't able to find anything when I went poking around. Thanks for the link!
